earthfile2llb/converter.go

```diff
@@ -1143,6 +1147,21 @@ func (c *Converter) Import(ctx context.Context, importStr, as string, isGlobal,
 	return c.varCollection.Imports().Add(importStr, as, isGlobal, currentlyPrivileged, allowPrivilegedFlag)
 }
 
+// Cache handles a `CACHE` command in a Target.
+// It appends run options to the Converter which will mount a cache volume in each successive `RUN` command.
+func (c *Converter) Cache(ctx context.Context, path string, isPersisted bool) error {
+	if err := c.checkAllowed(cacheCmd); err != nil {
+		return err
+	}
+	if isPersisted {
+		c.persistentCacheDirs = append(c.persistentCacheDirs, path)
```

Comment:
What happens if a user defines the same cache path twice?

Reply:
Good catch I think there would be risk of losing their data when we persist it. Will handle that more gracefully